ouroboros-network-framework/src/Ouroboros/Network/ConnectionManager/Core.hs

```diff
@@ -1187,13 +1191,13 @@ withConnectionManager ConnectionManagerArguments {
                         writeTVar connVar connState'
                         modifyTMVarPure_ stateVar $
                           (Map.update (\connVar' ->
-                            if eqTVar (Proxy :: Proxy m) connVar' connVar
-                               then Nothing
-                               else Just connVar')
-                          peerAddr)
+                                      if eqTVar (Proxy :: Proxy m) connVar' connVar
+                                          then Nothing
+                                          else Just connVar')
+                                      peerAddr)
```

Comment:
This change does not belong to this patch. I also prefer the previous one since it fits in 80 columns.